[2024-04-05 19:39:45,142] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados manual__2024-04-05T22:39:41.417391+00:00 [queued]>
[2024-04-05 19:39:45,146] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados manual__2024-04-05T22:39:41.417391+00:00 [queued]>
[2024-04-05 19:39:45,146] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:39:45,146] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-05 19:39:45,146] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:39:45,162] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-04-05 22:39:41.417391+00:00
[2024-04-05 19:39:45,164] {standard_task_runner.py:52} INFO - Started process 727058 to run task
[2024-04-05 19:39:45,166] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'manual__2024-04-05T22:39:41.417391+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/extrai_dados_climaticos.py', '--cfg-path', '/tmp/tmp4e60r127', '--error-file', '/tmp/tmp4akzftle']
[2024-04-05 19:39:45,167] {standard_task_runner.py:80} INFO - Job 132: Subtask extrai_dados
[2024-04-05 19:39:45,196] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados manual__2024-04-05T22:39:41.417391+00:00 [running]> on host TEC017860.
[2024-04-05 19:39:45,237] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-04-05T22:39:41.417391+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-05T22:39:41.417391+00:00
[2024-04-05 19:39:46,098] {python.py:173} INFO - Done. Returned value was: None
[2024-04-05 19:39:46,103] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20240405T223941, start_date=20240405T223945, end_date=20240405T223946
[2024-04-05 19:39:46,147] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-05 19:39:46,159] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
